lms/templates/courseware/courseware.html

```diff
@@ -89,31 +92,10 @@
   % endif
 
   <script type="text/javascript">
-    var $$course_id = "${course.id | escapejs}";
-
-    $(function(){
-        $(".ui-accordion-header a, .ui-accordion-content .subtitle-name").each(function() {
```

Comment:
This is definitely running on every page in the courseware. It seems to be cleaning up the titles of the items in the left hand accordion, and replacing spaces with ``&nbsp;`` presumably to stop the labels wrapping. This is a very ugly hack and it shouldn't live in this template, but I'd be nervous about removing it.